gitensite/apps/content/views.py

```diff
@@ -75,3 +77,32 @@ def post(self, request):
             return HttpResponse("OK")
         else:
             return HttpResponse("Incorrect key or key not present", status=401)
+
+class DownloadView(View):
+    def get(self, request, bookid):
+        results = Book.objects.filter(book_id=bookid)
```

Comment:
https://docs.djangoproject.com/en/2.0/topics/http/shortcuts/#get-object-or-404 may be useful here

Reply:
Thanks, this is a great shortcut! We updated the code to use it.